docker-compose.grafana.yaml

```diff
@@ -25,5 +25,5 @@ services:
         target: /var/lib/grafana/dashboards
     environment:
       - VIRTUAL_HOST=$DDEV_HOSTNAME
-      - HTTP_EXPOSE=3001:3000 # Grafana web interface via HTTP.
-      - HTTPS_EXPOSE=3000:3000 # Grafana web interface with HTTPS.
+      - HTTP_EXPOSE=3000:3000
+      - HTTPS_EXPOSE=3000:3000
```

Comment:
I don't believe you can ever do this on any system. I don't believe there are any HTTP servers that can handle HTTP and HTTPS on the same port.